corehq/messaging/scheduling/forms.py

```diff
@@ -216,9 +216,8 @@ def _clean_message_field(self, field_name):
                 raise ValidationError(_("This field is required"))
             return cleaned_value
 
-        for expected_language_code in self.schedule_form.language_list:
-            if not cleaned_value.get(expected_language_code):
-                raise ValidationError(_("Please fill out all translations"))
+        if not any(cleaned_value.values()):
+            raise ValidationError(_("Please fill out at least one translation"))
```

Comment:
It shouldn't, because `cleaned_value` only contains languages that are in the UI, which are the project's languages. There are some very edgy edge cases like opening this page and then adding/removing languages in a different tab.

I think the validation here is a nice to have and can be imperfect, ultimately the backend prevents blank messages from going out.